### PR TITLE
Added salt overdose

### PR DESCRIPTION
**Pull request by Cruix**
True salt transcends the fourth wall.
:cl: 
rscadd: You can now overdose on salt.
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
This is the best PR yet

**Comment by ShadowDeath6** [comment]
>not adding the lines complaining about nerfs
>not adding "fuck off if ur not headcoder"

gppa

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -109,4 +109,47 @@
...
+		"SILICON"
+	]
+	
+	"salt": [
+        "MaRGH MaH PRs!",
+        "FUCKING SHITSEC",
+        "THIS IS AGAINS SPESS LAW",
+        "ReBase TO @pick(CODEBASE)",
+        "REVIRT TO OLD @pick(CODEBASE)",
+		"AI WONT OBEY LAW 2 ITS ROGUE!!",
```
Comment (oisin100) [inline strings/brain_damage_lines.json:155]: AAAAAAAAAAARGH muh indentation

**Review by oisin100 — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -246,13 +246,23 @@
...
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
 
+/datum/reagent/consumable/sodiumchloride/overdose_process(mob/living/M)
+	if(prob(saltprob))
+		var/line = pick_list_replacements(BRAIN_DAMAGE_FILE, "salt")
+		if(prob(radioprob))
+			line = ";" + line
+		M.say(line)
+
```
Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm:268]: add a prob to emote cry or whine as well

File: strings/brain_damage_lines.json (hunk 1 above)
Comment (Cruix) [reply strings/brain_damage_lines.json:155]: What?

File: strings/brain_damage_lines.json [hunk 3]
```diff
@@ -109,4 +109,47 @@
...
+		"LeAVE YuR AUTISM fORt sO i CAN kILL yOU",
```
Comment (Super3222) [inline strings/brain_damage_lines.json]: LMAO

**Comment by Kmc2000** [comment]
Add in:

WHAT THE FUCK ESWORD CAN KNOCK U OUT INSTANT STUNS BROKEN GAME GGG

File: strings/brain_damage_lines.json (hunk 1 above)
Comment (oisin100) [reply strings/brain_damage_lines.json:155]: You're using tabs to indent in some places and spaces in others http://prntscr.com/dqpuss

**Comment by Jc1469** [comment]
REVERT NINJA CHANGE

File: strings/brain_damage_lines.json [hunk 4]
```diff
@@ -1,112 +1,178 @@
...
+		"no rp",
+	]
+	
+	"DEPARTMENT": [
+		"BOTANY",
+		"TRAITOR",
+		"SCIENCE",
+		"CHEMISTRY",
+		"SECURITY",
+		"CLOCKCULT",
```
Comment (Time-Green) [inline strings/brain_damage_lines.json:135]: COCKCULT

File: strings/brain_damage_lines.json [hunk 5]
```diff
@@ -1,112 +1,178 @@
...
+		"Freezing me is breaking rp",
+		"I DEMAND YOU BAN HIM",
+		"fUKKING cLoWN Is A GrIEFFeR",
+		"CODERZ CONSIRIN AGAINST MEH",
+		"Muh Slippery SLope",
+		"NOONE RPS",
+		"RP IS FOR DUMBSHITS",
+		"AI SI RACIST AGIANS PLSAMAMEN",
+		"HES OBIVOUSLY A TATOR",
+		"AAAAAAAAAAARGH muh indentation",
```
Comment (oisin100) [inline strings/brain_damage_lines.json:167]: 👍

**Review by oisin100 — APPROVED** [review]

File: strings/brain_damage_lines.json [hunk 6]
```diff
@@ -127,29 +135,44 @@
...
 		"CODERZ CONSIRIN AGAINST MEH",
-		"@pick(DEPARTMENT) POWERCREPE",
 		"Muh Slippery SLope",
 		"NOONE RPS",
 		"RP IS FOR DUMBSHITS",
 		"AI SI RACIST AGIANS PLSAMAMEN",
-		"HES OBIVOUSLY A TATOR"
-    ]
+		"HES OBIVOUSLY A TATOR",
+		"AAAAAAAAAAARGH muh indentation",
```
Comment (ShadowDeath6) [inline strings/brain_damage_lines.json]: oisin btfo

**Pull request merged**